tests/Behat/features/edit-block-element.feature

```diff
@@ -32,5 +32,7 @@ Feature: Edit elements in the CMS
       And I press the "Publish" button
     Then I should see a "Published content block" message
     When I go to "/admin/pages/edit/show/6"
-      Then I should see "Eve's Block"
+      And I wait until I see the ".element-editor__element" element
```

Comment:
I think I mentioned this on another PR - this assertion isn't designed to select a class - you're basically saying "wait until I see the [any element] element" where it's designed to be more specific than that e.g. "wait until I see the [specific] element".

We could add another step like "wait until the elements have loaded" instead maybe?